tests/acceptance/30_generic_methods/classes_copy.cf

```diff
@@ -37,9 +37,10 @@ bundle agent test
     "ph2" usebundle => file_ensure_lines_absent("${init.file}", "sample one"); # should get success
     "ph3" usebundle => file_ensure_lines_absent("/dev/zero", "sample three"); # should get failed
 
-    "ph1"  usebundle => _classes_copy("file_ensure_lines_absent_${init.file_canon}","global_classA");
     "ph2"  usebundle => _classes_copy("file_ensure_lines_present_${init.file_canon}","global_classB");
```

Comment:
Because the original ordering made no sense. Now, the order matches the order of the calls above.